authentication/wildfly/configure-server.cli

```diff
@@ -0,0 +1,5 @@
+embed-server --admin-only=true
+/subsystem=logging/logger=org.wildfly.security:add(level=TRACE)
+/subsystem=undertow/application-security-domain=other:write-attribute(name=integrated-jaspi, value=false)
+/subsystem=elytron/policy=jacc:add(jacc-policy={})
```

Comment:
This should first check if the logger exists so the server can be reused without being cleaned first.

Reply:
As above at the moment these scripts only run if we did not already have a server installed.